api/serializers.py

```diff
@@ -34,8 +36,8 @@ class DocumentSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
             model = Document
             fields = (
-                'title', 
-                'slug', 
+                'title',
```

Comment:
these are all removed automatically from my IDE. I managed to ignore them later, but missed them in these early commits. Let me know if you want me to bring them back.

Reply:
Huh, I can't imagine why it removes them, maybe they're unnecessary?

Let's try it out in Staging and see what happens. :)

Reply:
oh it's not clear from the other diff, but I just was talking about the trailing spaces. there seems to be a handful around other files too.